core-bundle/src/Command/MigrateCommand.php

```diff
@@ -77,8 +77,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->io = new SymfonyStyle($input, $output);
 
-        if (!$input->getOption('dry-run') && !$input->getOption('no-backup')) {
-            $this->backup($input);
+        if (!$input->getOption('dry-run') && !$input->getOption('no-backup') && !$this->backup($input)) {
+            return 1;
```

Comment:
Note: needs to be changed to `return Command::FAILURE;` when merged upstream, cc @leofeyer